### DISCUSSION TITLE
How to update

### DISCUSSION THREAD

**Discussion by TanaKa1990** [Intermesh/groupoffice#1035] [Q&A]
Hi,

I am absolutely new to group office and would like to know, how can I update my self hosted installation of group office.
I must say, that I am not so involved with these kind of self hosted apps. I have installed the GO app by "one click install" and not more details how can I run bash scripts, package and so on...

Can you explain me, how I can update my GO App on my hosted server ?
   
Tank you very much.

**Comment by mschering** [comment]
Hi,

How did you install? It sounds like Softaculous or Installatron? They lag behind in updates unfortunately. You will have to wait for them to update it or upgrade it using an alternative install method. We could also offer to help you with that.

Best regards,
Merijn

**Reply by TanaKa1990** [reply]
Hi,

My hoster offers a selection of apps to install as a one-click installation. 

BR

Taner

**Reply by mschering** [reply]
Sorry, you are dependent on the hosting provider. Which version are you currently on? We could offer to help you with installing an update with another method.
